app/assets/stylesheets/pages/_home.scss

```diff
@@ -2,3 +2,57 @@
  * CSS code specific to home page
  * (margin/padding adjustments)
  * ------------------------------------- */
+ .banner {
+  background: center no-repeat;
+  background-size: cover !important;
+  position: relative;
+  color: white;
+  text-align: center;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Source Sans Pro', sans-serif;
```

Comment:
also for the font-family may be we can set it in the variables SCSS, could you check that ❓